master/docs/developer/stats-service.rst

```diff
@@ -68,44 +70,68 @@ Stats Service
 Storage backends
 ----------------
 
-Storage backends are responsible for storing any stats-data sent to them.
+Storage backends are responsible for storing any statistics/data sent to them.
 A storage backend will generally be some sort of a database-server running on a machine.
-(*Note*: This machine may be different from the one running :class:`BuildMaster`)
+.. note:: This machine may be different from the one running :class:`BuildMaster`
 
 Data is captured according to the master config file and then, is sent to each of the storage backends provided by the master configuration (see :bb:cfg:`stats-service`).
 
-Each storage backend has a Python client defined as part of :mod:`buildbot.statistics.storage_backends`
-to aid in posting data by :class:`StatsService`
+Each storage backend has a Python client defined as part of :mod:`buildbot.statistics.storage_backends` to aid in posting data by :class:`StatsService`
 
 Currently, only `InfluxDB <http://influxdb.com>`_ is supported as a storage backend.
 
-.. py:class:: StatsStorageBase
+.. py:class:: buildbot.statistis.storage_backends.base.StatsStorageBase
 
-   A base class for all storage services
+   A abstract base class for all storage services. It cannot be directly initialized - it would raise a ``TypeError`` otherwise.
 
-.. py:class:: InfluxStorageService
+   .. py:method:: thd_postStatsValue(self, post_data, series_name, context={})
+
+      :param post_data: A dict of key-value pairs that is sent for storage.
```

Comment:
Let's be consistent about parameter documentation.

Though Sphinx offers this way to document parameters, it takes way too much space.